test/text/ttml_text_parser_unit.js

```diff
@@ -57,32 +57,33 @@ describe('TtmlTextParser', () => {
         '<tt xml:space="default">' + ttBody + '</tt>',
         {periodStart: 0, segmentStart: 60, segmentEnd: 70, vttOffset: 0},
         {startTime: 62.03, endTime: 62.05});
+    // NOTE: This cannot be supported (easily) using the tXml parser.
```

Comment:
What is the explanation for this? Do you know what dash.js does in this case when using txml?

Reply:
And can we make it similar here?

Reply:
I could add feature to our implementation of tXml to do something similar to this when parsing tt elements.

Reply:
Please